### PR TITLE
Mr 27 contribution plot

### PR DESCRIPTION
**Pull request by JanionNevill**
Created indirect contribution plot and composite regression plot.

After the changes have been reviewed, but before the branch is merged, I'll move some files around to fix the file structure. No code changes will be made at that stage.

### CODE REVIEW

File: R/meta_regression_plots.R [hunk 1]
```diff
@@ -14,6 +15,12 @@
 #' @param confidence_opacity The opacity of the confidence regions. Can be any value between 0 and 1, inclusive. Defaults to 0.2.
 #' @param include_contribution TRUE if study contribution should be displayed as circles. Defaults to TRUE.
 #' @param contribution_multiplier Multiplication factor by which to scale the sizes of the study contribution circles. Defaults to 1.0.
+#' @param legend_position String informing the position of the legend. Acceptable values are:
+#' - "BR" - Bottom-right of the plot area
+#' - "BL" - Bottom-left of the plot area
+#' - "TR" - Top-right of the plot area
+#' - "TL" - Top-left of the plot area
+#' Defaultsa to "BR"
```
Comment (nabury) [inline R/meta_regression_plots.R:23]: Typo

**Review by nabury — CHANGES_REQUESTED** [review]
Just one minor request to add a comment where it's not obvious to me what the code is doing. Looks good!

File: R/meta_regression/composite_regression_plot.R [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    model = model,
+    treatment_df = treatment_df,
+    reference = reference,
+    comparators = comparators,
+    covariate_value = covariate_value,
+    contribution_type = contribution_type,
+    include_ghosts = include_ghosts,
+    contribution_multiplier = contribution_multiplier
+  )
+
```
[the remark this answers is not in the document]
Comment (JanionNevill) [reply R/meta_regression/composite_regression_plot.R:70]: Added

File: R/meta_regression_plots.R (hunk 1 above)
Comment (JanionNevill) [reply R/meta_regression_plots.R]: Good spot. Fixed.

**Review by nabury — APPROVED** [review]

**Pull request merged**